Adds new Meetup Organizers Toolkit (#9)
ansible-community/meetup: milestones/during_the_event.md

## Guidelines

> Now the Big day is here, the day of the event. And you are all ready for it 🙂. Now here are the Guidelines to follow  during the day of the meetup. These are some steps that, if you follow it, will make your life easier as an organizer.

Reviewer: ```suggestion
Now the big day is here, the day of the event. And you are all ready for it 🙂. Here are some guidelines for the day of the meetup that can make your life easier as an organizer.
```

**Strict Guidelines****

> 1. Follow the Ansible Community Code of Conduct at all times during the event.

Reviewer 2: We probably want to mention the Code of Conduct (and link to it!) on the main Meetup Toolkit page. If I've organized a meetup, I don't want the "Big Day" to be the first time I'm thinking about the Code of Conduct.

Author: Add this 'Follow the Ansible Community Code of Conduct at all times during the event. Read about the Code of Conduct in the [resource section](./resources/ansible_community_code_of_conduct.md) prior the event. It helps the organizer to understand the ethos of the Ansible Community.'

> 2. Be Inclusive

Reviewer 2: ```suggestion
2. Be Inclusive.
```

Author: Updated.

> 3. Take care of the Security concerns.

Reviewer: Do we provide something that spells out the security concerns?

Author: That is a good idea, to be precise or just some ideas may be. Can you please share some ideas?

4. Make sure to announce the name of the person of contact in case of CoC violation.
5. Mode of communication in case of CoC complaints.
6. Make sure that you have a First Aid kit ready.

> 7. Ask for permission of the attendees, speakers and volunteers  to be photographed.

Reviewer: ```suggestion
7. Ask all attendees, speakers, and volunteers for photography permissions.
```

Author: Updated.

> 8. Ask for permission of the attendees, speakers and volunteers  to be tagged in Social Media.

Reviewer: ```suggestion
8. Ask all attendees, speakers and volunteers for permission to be tagged or mentioned on social media.
```

**Good to follow Guidelines

1. Time Management

> Start and end the event on time.Ensure speakers also keep to their time allowance, especially if you have multiple speakers - this is respectful to people's time, especially the latter speakers in the agenda.

Reviewer 2: ```suggestion
Start and end the event on time. Ensure speakers also keep to their time allowance, especially if you have multiple speakers - this is respectful to people's time, especially the latter speakers in the agenda.
```

Author: updated.

2. Round table introduction

> A round table introduction before the meetup helps us to create a network with the attendees. This helps the organizers to know them, their work and experience level better. Many times you might get your future speaker, organizer or volunteer for the group from there. Therefore be frank in asking for help.

Reviewer: ```suggestion
A round table introduction before the meetup helps us to create a network with the attendees. This helps organizers get to know attendees and their work and experience levels. You can often find future speakers, organizers, or volunteers among the event attendees so it is always worth asking for help.
```

Author: Updated.

3. Feedback session

> An honest and frank feedback session at the end of the meetup is essential. Encourage the participants to comment on the meetup page or fill up the meetup.com feedback form for giving their feedback.You can ask the following questions in there :

Reviewer 2: ```suggestion
An honest and frank feedback session at the end of the meetup is essential. Encourage the participants to comment on the meetup page or fill up the meetup.com feedback form for giving their feedback. You can ask the following questions in there :
```

Author: Updated.

- How was the event ?
- Was it helpful for the attedees?
- How can the meetup improve?

> - What are the topics you would like to hear more about in future sessions?

Reviewer 2: ```suggestion
- Do you have a topic you could present in a future session?
- Do you have a topic you'd like someone else to present in a future session?
```

5. Fix the plan for the next session

> It is advisable to plan the next event, the date, venue and brief agenda at the end of the meetup. So the attendees leave  with a definitive date to be booked on their calendar. Also it gives them some idea where the community is heading. Once you make the plan with the attendees they feel more inclusive. Giving their wishes priority  you essentially make them feel a part of the group itself.

Reviewer: ```suggestion
It is advisable to plan the next event, the date, venue and brief agenda at the end of the meetup. So the attendees leave with a definite date to book on their calendars. Also it gives them some idea where the community is heading. Once you make the plan with the attendees they gain a sense of inclusion. Giving their wishes priority you essentially make them feel a part of the group itself.
```

6. How to connect with the Ansible Community?

Encourage the attendees to join the Ansible Community [Matrix channel](https://docs.ansible.com/ansible/latest/community/communication.html#general-channels) and show them how to join Matrix if they are not there.
Join the Ansible Community Matrix space, find a regional channel they can connect in. You may start a private Matrix room with the attendees to be connected.

## Presentation Templates

There are some presentation templates in the Resources Section below to help you with the presentation ideas and giving those ideas the form of presentation.

> Past Talks

Reviewer: ```suggestion

## Past Talks
```
